src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionpostponementrequest/ActionPostponementRequestAboutToSubmitHandlerTest.java

```diff
@@ -169,7 +169,7 @@ public void givenAGrantedPostponementAndReadyToList_thenClearReviewStateAndRefer
         assertThat(response.getData().getState(), is(State.READY_TO_LIST));
         assertThat(response.getData().getSscsDocument(), is(not(empty())));
         assertThat(response.getData().getPostponementRequest().getUnprocessedPostponementRequest(), is(YesNo.NO));
-        assertThat(response.getData().getDwpState(), is(DwpState.DIRECTION_ACTION_REQUIRED.getId()));
```

Comment:
I think this needs to go back to business  + POs to confirm that we're changing legacy behaviour

Reply:
@dattipoeHMCTS checked with BA (Makeda) and confirmed this behaviour change.